ra/registration-authority_test.go

```diff
@@ -748,11 +748,15 @@ func TestAuthzRateLimiting(t *testing.T) {
 			Threshold: 1,
 			Window:    cmd.ConfigDuration{Duration: 24 * 90 * time.Hour},
 		},
+		ValidAuthorizationsPerName: cmd.RateLimitPolicy{
```

Comment:
Rather than `Name`, can we call this `ValidAuthorizationsPerFQDN`? It's more specific, and matches the other FQDN rate limit. We use `Name`, somewhat ambiguously, in other rate limits to mean a parent domain.